lib/node_modules/@stdlib/stats/base/dmskrange/lib/ndarray.native.js

```diff
@@ -42,27 +40,16 @@ var addon = require( './dmskrange.native.js' );
 * @example
 * var Float64Array = require( '@stdlib/array/float64' );
 * var Uint8Array = require( '@stdlib/array/uint8' );
-* var floor = require( '@stdlib/math/base/special/floor' );
 *
 * var x = new Float64Array( [ 2.0, 1.0, 2.0, -2.0, -2.0, 2.0, 3.0, 4.0, 5.0, 6.0 ] );
 * var mask = new Uint8Array( [ 0, 0, 0, 0, 0, 0, 0, 0, 1, 1 ] );
-* var N = floor( x.length / 2 );
 *
-* var v = dmskrange( N, x, 2, 1, mask, 2, 1 );
+* var v = dmskrange( 5, x, 2, 1, mask, 2, 1 );
 * // returns 6.0
 */
 function dmskrange( N, x, strideX, offsetX, mask, strideMask, offsetMask ) {
-	var viewX;
-	var viewM;
-	if ( strideX < 0 ) {
-		offsetX += (N-1) * strideX;
-	}
-	if ( strideMask < 0 ) {
-		offsetMask += (N-1) * strideMask;
-	}
-	viewX = new Float64Array( x.buffer, x.byteOffset+(x.BYTES_PER_ELEMENT*offsetX), x.length-offsetX ); // eslint-disable-line max-len
-	viewM = new Uint8Array( mask.buffer, mask.byteOffset+(mask.BYTES_PER_ELEMENT*offsetMask), mask.length-offsetMask ); // eslint-disable-line max-len
-	return addon( N, viewX, strideX, viewM, strideMask );
+	// TODO - add eslint comment to escape max line length warning(if needed)
+	return addon.ndarray( N, x, strideX, offsetX, mask, strideMask, offsetMask );
```

Comment:
@aayush0325 The necessary comment was present in the lines you removed. See above in the diff.

```suggestion
	return addon.ndarray( N, x, strideX, offsetX, mask, strideMask, offsetMask ); // eslint-disable-line max-len
```

Reply:
ah, i'll just push this wherever it is needed. thanks for pointing it out